reana_workflow_controller/rest/workflows.py

```diff
@@ -409,6 +411,7 @@ def create_workflow():  # noqa
             logs="",
             git_ref=git_ref,
             git_repo=git_repo,
+            workspace_path=build_workspace_path(request.args["user"], workflow_uuid),
```

Comment:
I guess this line is here for testing purposes? I think it should be better to leave this part as it was before, since `build_workspace_path()` will be called during workflow initialization from `reana-db models`